tests/e2e/test_parameterized_queries.py

```diff
@@ -0,0 +1,382 @@
+import datetime
```

Comment:
Most of the contents of this file were moved from `parameterized_query_tests.py`. I renamed the file because pytest won't discover it unless the filename begins with `test_`.